### PR TITLE
test: updated file-upload-item screenshots

### PR DESCRIPTION
**Pull request by SiebenSieben**

### CODE REVIEW

**Review by reme3d2y — APPROVED** [review]

**Review by SiebenSieben — COMMENTED** [review]
Test comment

File: packages/file-upload-item/src/__image_snapshots__/file-upload-item-hide-meta-when-upload-status-success-name-0-upload-date-0-size-0-show-delete-0-upload-status-0-snap.png
Comment (SiebenSieben) [inline packages/file-upload-item/src/__image_snapshots__/file-upload-item-hide-meta-when-upload-status-success-name-0-upload-date-0-size-0-show-delete-0-upload-status-0-snap.png]: Test comment #7

Comment (SiebenSieben) [inline packages/file-upload-item/src/__image_snapshots__/file-upload-item-hide-meta-when-upload-status-success-name-0-upload-date-0-size-0-show-delete-0-upload-status-0-snap.png]: Test

Comment (SiebenSieben) [inline packages/file-upload-item/src/__image_snapshots__/file-upload-item-hide-meta-when-upload-status-success-name-0-upload-date-0-size-0-show-delete-0-upload-status-0-snap.png:1]: Yet another test haha

Comment (Oladii) [inline packages/file-upload-item/src/__image_snapshots__/file-upload-item-hide-meta-when-upload-status-success-name-0-upload-date-0-size-0-show-delete-0-upload-status-0-snap.png:1]: тест2

**Pull request merged**